### ISSUE TITLE
Migrate gRPC to the new ASP.NET Core v3 gRPC

### ISSUE THREAD

**Issue by ankitkumarr** [Azure/azure-functions-host#5449]

**Comment by ankitkumarr** [comment]
The initial changes are done, and works fine locally. But we are facing issues with Sandbox restrictions in Azure. In particular, sandbox seems to be blocking language worker processes to connect to the localhost via HTTP/2.

This will likely move to the next Sprint while we work through those hurdles.

**Comment by ankitkumarr** [comment]
This will need to move to the next Sprint.

I didn't get to spend too much time with this. But I have collected the data I need to start the conversation with our internal teams. I spend some time validating the issue by adding logs and such to make sure the all ports are opened correctly and this is not an issue with our changes.

The issue is particularly interesting because a process connecting to a local port opened by another process is a supported feature of sandbox and this is how custom handlers and grpc works until now.

**Comment by fabiocav** [comment]
Additional investigation is needed here to identify some sandbox issues.

**Comment by ankitkumarr** [comment]
Moving this to Sprint 99 for landing the changes perhaps in Linux (where sandbox is not applicable) in the meanwhile.

**Comment by fabiocav** [comment]
This is, again, not going to make the cut for planning. So moving this to the following sprint.

**Comment by fabiocav** [comment]
Work on this is still in progress. We're still blocked by platform issues on both, Windows and Linux. 

@brettsam is driving the investigation and we'll continue to update this issue as we make progress.

Flagging @pragnagopa and @pgombar as this is likely related to another issue they're currently investigating.

**Comment by fabiocav** [comment]
This is still blocked. @brettsam is currently driving the investigation.

**Comment by fabiocav** [comment]
We're now unblocked. 

@brettsam is working through final updates and test failures and we should be merging this in 110.

**Comment by fabiocav** [comment]
Closing as done. 🎉 🦖
